arkouda/client.py

```diff
@@ -207,6 +207,8 @@ def connect(
     regexMaxCaptures = serverConfig["regexMaxCaptures"]  # type:ignore
     clientLogger.info(return_message)
 
+    return cast(str, generic_msg(cmd="getconfig", args={}))
```

Comment:
I don't think we need to return the config info here. The use case provided, this will be read from `stdout` when the server is started.

Reply:
Removed - I added this initially as a free convenience bit